package.json

```diff
@@ -100,6 +100,7 @@
     "zone.js": "^0.7.7"
   },
   "dependencies": {
-    "date-fns": "^1.28.0"
+    "date-fns": "^1.28.0",
+    "lodash": "^4.17.4"
```

Comment:
With NPM I don't think it matters that much tbh :) But I changed in a new commit :+1: